src/main/kotlin/com/markskelton/settings/ThemeSettingsUI.kt

```diff
@@ -0,0 +1,132 @@
+package com.markskelton.settings
+
+import com.intellij.ide.BrowserUtil.browse
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.options.SearchableConfigurable
+import com.intellij.openapi.project.DumbAware
+import com.intellij.openapi.ui.DialogPanel
+import com.intellij.ui.layout.panel
+import com.markskelton.settings.ThemeSettings.Companion.constructSettingModel
+import java.net.URI
+import javax.swing.JComponent
+
+data class ThemeSettingsModel(
+  var isBold: Boolean,
+  var isVivid: Boolean,
+  var isItalic: Boolean
+)
+
+class ThemeSettingsUI : DumbAware, SearchableConfigurable {
+
+  companion object {
+    const val THEME_SETTINGS_DISPLAY_NAME = "One Dark Theme Settings"
+    private const val REPOSITORY = "https://github.com/one-dark/jetbrains-one-dark-theme"
+    private val CHANGELOG_URI = URI("$REPOSITORY/blob/master/CHANGELOG.md")
```

Comment:
This will jump them directly to the changelog header.

```suggestion
    private val CHANGELOG_URI = URI("$REPOSITORY/blob/master/CHANGELOG.md#changelog")
```